modules/claims_api/lib/claims_api/error/error_handler.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'claims_api/common/exceptions/token_validation_error'
+
+module ClaimsApi
+  module Error
+    module ErrorHandler
+      def self.included(clazz)
+        clazz.class_eval do
+          rescue_from ::Common::Exceptions::TokenValidationError,
+                      with: lambda {
+                        render_error(ClaimsApi::Error::TokenValidationError.new)
+                      }
+          rescue_from ::Common::Exceptions::ResourceNotFound,
+                      ::Common::Exceptions::Forbidden,
+                      ::Common::Exceptions::Unauthorized,
+                      ::Common::Exceptions::ValidationErrorsBadRequest,
+                      ::Common::Exceptions::UnprocessableEntity do |err|
+                        render_error(err)
+                      end
```

Comment:
This ticket has lingered on for too long, but I think we should come back at some point and follow the pattern shown on lines10-13.
Create custom exception classes for each of these and follow that pattern.
We're going to have to do this if we decide someday that we need to introduce some kind of error sub-code mechanism.